tests/run_integration_tests.sh

```diff
@@ -100,17 +100,16 @@ for t in test_*; do
     res=0
     # run the test scripts
     for f in scripts/*.[Rr]ev ; do
-        ${rb_exec} -b $f # print output so we can see any error messages
+        mkdir output
+        tmp0=${f#scripts/}
+        tmp1=${tmp0%.[Rr]ev}
+        ${rb_exec} -b $f &> output/${tmp1}.errout # print output so we can see any error messages
```

Comment:
What is the reasoning for combining the stderr and the stdout into a single file?
It seems better to separate these files.
That would allow checking one but not the other.

Reply:
Is that sort of flexibility really needed?

Reply:
If you want to check that the error is written to the right stream (i.e. stderr and not stdout), then it is needed.  If you look at other test suites like this, they typically do not combine the stderr and the stdout.

But I guess writing stderr and stdout separately loses the interleaving, which might also be important.  Ideally we could write `name.stderr`, `name.stdout`, and `name.errout`.  But this turns out to be a bit tricky.  So I think this is fine for now.